### ISSUE TITLE
main.py does not run in Blender

### ISSUE THREAD

**Issue by pedromartin5000** [AdrianJMartin/Blender-2-Minecraft#1]
When I "run the script" in Blender, I get the following message...

"Python script failed, check the message in the system console"

...I don't see any message in the console.

![image](https://user-images.githubusercontent.com/96946163/147852333-a4f36434-821b-49f1-b8dc-33f74b745dba.png)

The only change I have made is the value of output_filepath

**Comment by AdrianJMartin** [comment]
Is it you own scene? Have you deleted the camera and the light? I may forgotten to mention that in the readme!

And what does the console say - you either need to start B from the cmd line or **Window Menu -> Toggle System Console**.

**Comment by AdrianJMartin** [comment]
The codes changed a bit - I need to push!  but something like changing line 40 to 

    if o.name.startswith("mc-cube") is False and o.type == 'MESH':

**Comment by pedromartin5000** [comment]
Ah, I started with my own scene.
So, I need to delete the camera and the light and add the 'mc-cube' and 'mc-cube-limit'?
I'll start with you sample scene for now...

**Comment by AdrianJMartin** [comment]
yeah... In that version of the script you don't need the limit one - its hard coded in the script..

**Comment by AdrianJMartin** [comment]
I'm about to do a push of the stuff - I've been working on. ...Its a bit different as I'm focusing on doing blocks first - and then curved/scanned objects...Its going to require building your scene with a few more rules....but its still way easier to build than individual blocks inside mc...
